docs/configuration.asciidoc

```diff
@@ -857,14 +851,14 @@ The unit is provided as suffix directly after the number, without any separation
  * `mb` (megabytes)
  * `gb` (gigabytes)
 
-NOTE: we use the power-of-two sizing convention, e.g. `1 kilobyte == 1024 bytes`
+NOTE: We use the power-of-two sizing convention, e.g. `1 kilobyte == 1024 bytes`.
 
 [float]
 [[special-configuration]]
 === Special configuration
 
-Elastic APM patches `Kernel#require` to auto-detect and instrument supported third party libraries. It does so with the utmost care but in rare cases it can conflict with some libraries.
+Elastic APM patches `Kernel#require` to auto-detect and instrument supported third party libraries. However, it may conflict with some libraries.
```

Comment:
```suggestion
Elastic APM patches `Kernel#require` to auto-detect and instrument supported third party libraries. It does so with the utmost care but in rare cases it can conflict with some libraries.
```